### PR TITLE
Update vdf.php

### PR DESCRIPTION
**Pull request by Shmurdik**
sprintf format for correct pretty level

### CODE REVIEW

File: vdf.php [hunk 1]
```diff
@@ -130,13 +130,13 @@ function vdf_encode_step($arr, $pretty, $level) {
...
         if(is_string($v)) {
-            $buf .= sprintf('%s"%s" "%s"\n', $line_indent, $k, $v);
+            $buf .= sprintf("%s\"%s\" \"%s\"\n", $line_indent, $k, $v);
         }
         else {
             $res = vdf_encode_step($v, $pretty, $level + 1);
             if($res === NULL) return NULL;
 
-            $buf .= sprintf('%s"%s"\n{\n%s}\n', $line_indent, $k, $res);
+            $buf .= sprintf("%1\$s\"%2\$s\"\n%1\$s{\n%3\$s%1\$s}\n", $line_indent, $k, $res);
```
Comment (rossengeorgiev) [inline vdf.php]: Use single quoted literals here and there won't be a need to escape `$` or `"`

**Comment by Shmurdik** [comment]
in single quotes "\n" written as simple text, but not as special symbol

**Comment by rossengeorgiev** [comment]
Ah yes.... well

**Comment by rossengeorgiev** [comment]
Fix #11

**Pull request merged**